File: MCA/First Year/second Sem/LAB Activity/DSA Lab/DataStructure/Array.c
#include <stdio.h>
int main()
{
    int n;
    // get the size of array
    printf("Enter the size of Array\n");
    scanf("%d", &n);
    int arr[n];
    // get element of array
    printf("Enter the Element of Array\n");
    for (int i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }
    // Print the array
    printf("Array's Element is:\n");
    for (int i = 0; i < n; i++)
    {
        printf("%d ", arr[i]);
    }
    return 0;
}

File: Placement/DSA JAVA/Array/Array.java
import java.util.Scanner;

public class Array {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the size of array");
        int size = sc.nextInt();
        int[] array = new int[size];
        System.out.println("Enter the Element of array");
        for (int i = 0; i < size; i++) {
            array[i] = sc.nextInt();
        }
        System.out.println("The element of array is");
        for (int num : array) {

            System.out.print(num+" ");
        }
        sc.close();
    }
}